standard/json.lua

```diff
@@ -0,0 +1,38 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Json
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local json = {}
+
+local getArgs = require('Module:Arguments').getArgs
+
+function json.fromArgs(frame)
+	local args = getArgs(frame)
+	return json.stringify(args)
+end
+
+function json.stringify(obj, pretty)
+	return mw.text.jsonEncode(obj, pretty == true and mw.text.JSON_PRETTY or nil)
+end
+
+function json.parse(obj)
+	local parse = function(object) return mw.text.jsonDecode(object, mw.text.JSON_TRY_FIXING) end
+	local status, res = pcall(parse, obj);
+	if status then
+		return res, false
+	else
+		mw.log('Error: could not parse Json:')
+		mw.logObject(obj)
```

Comment:
```suggestion
		mw.logObject(obj, 'Error. Could not parse JSON')
```

Reply:
Even better if the function could accept an optional argument to allow specifying the source of this error.